packages/css/src/grid/grid-cell.scss

```diff
@@ -5,14 +5,46 @@
 
 @import "../../utils/breakpoint";
 
-@media screen and (width > $amsterdam-breakpoint) {
-  @for $i from 1 through 12 {
-    .amsterdam-grid-column-start-#{$i} {
+.amsterdam-page-grid > .amsterdam-grid-cell-full-width {
+  grid-column: 1 / -1;
+}
+
+@for $i from 1 through 12 {
```

Comment:
Probably my personal opinion but loops should stay out of css, I would've typed it out..